ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -261,15 +261,20 @@ let on_request :
       | _ -> None
     in
 
-    let format_contents ~as_markdown ~typ ~doc =
+    let format_contents ~syntax ~as_markdown ~typ ~doc =
+      let languageId = 
+        match syntax with
+        | Document.Syntax.Ocaml -> "ocaml"
+        | Document.Syntax.Reason -> "reason"
```

Comment:
This is a detail, but this is the inverse function of `Document.of_language_id`, would you mind moving it in `document.ml` and name it `to_language_id` ?

Reply:
Sure thing! Thanks for the feedback.

Reply:
Added in https://github.com/ocaml/ocaml-lsp/pull/207/commits/d83470a129dbc13827bd98820d912005e51ce061